src/components/svg/LogoText.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const LogoText = () => (
+  <svg width="100%" height="100%" viewBox="0 0 581 82" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
```

Comment:
This needs Props to be passed down like the deleted HNScan.js component

Reply:
updated with props and theming